.flattened-pom.xml

```diff
@@ -8,15 +8,31 @@
   <packaging>bundle</packaging>
```

Comment:
.flattened have to be ignored

Reply:
[90fb16d](https://github.com/BlackBeltTechnology/karaf-jasypt-support/pull/2/commits/90fb16dae3be9705ff56e77a2b4bc60309c32233)